x-pack/plugins/security_solution_serverless/public/plugin.ts

```diff
@@ -69,10 +77,20 @@ export class SecuritySolutionServerlessPlugin
     registerUpsellings(securitySolution.getUpselling(), productTypes, services);
 
     securitySolution.setComponents({
-      GetStarted: getSecurityGetStartedComponent(services, productTypes),
       DashboardsLandingCallout: getDashboardsLandingCallout(services),
     });
-
+    securitySolution.setGetStartedPageSettings.setProductTypes(productTypes);
+    securitySolution.setGetStartedPageSettings.setProjectFeaturesUrl(
+      getProjectFeaturesUrl(services.cloud)
+    );
+    securitySolution.setGetStartedPageSettings.setAvailableSteps([
+      CreateProjectSteps.createFirstProject,
+      OverviewSteps.getToKnowElasticSecurity,
+      AddIntegrationsSteps.connectToDataSources,
+      ViewDashboardSteps.analyzeData,
+      EnablePrebuiltRulesSteps.enablePrebuiltRules,
+      ViewAlertsSteps.viewAlerts,
+    ]);
```

Comment:
Consider extracting these calls to a `setOnboardingSettings(services);`, so we keep the plugin `start` simple and declarative.